vms/volunteer/templates/volunteer/edit.html

```diff
@@ -243,12 +245,13 @@
                         </div>
                     </div>
                 {% else %}
-                    <div class="form-group">
+                    <div id="div_id_unlisted_organization" class="form-group">
                         <label class="col-md-2 control-label">{% trans "If your organization is not listed, please provide it here:" %}</label>
                         <div class="col-md-8">
-                            <input class="form-control" type="text" placeholder="{% blocktrans %}Organization{% endblocktrans %}" name="unlisted_organization" value="{% if form.unlisted_organization.value %}{{ form.unlisted_organization.value }}{% endif %}">
+                            <input id="id_unlisted_organization" class="form-control" type="text" placeholder="{% blocktrans %}Organization{% endblocktrans %}" name="unlisted_organization" value="{% if form.unlisted_organization.value %}{{ form.unlisted_organization.value }}{% endif %}">
                         </div>
                     </div>
+                <script type="text/javascript" src="{% static 'vms/js/hide_organization.js' %}"></script>                 

```

Comment:
is this a new file you are introducing ? Please commit the one to app/static/ or media/ ?

Reply:
No @tonythomas01 Its already there.

Reply:
cool then.